extensions/amp-lightbox/amp-lightbox.md

```diff
@@ -164,6 +169,8 @@ In the examples below, we demonstrate how the transition looks for an AMPHTML ad
 </noscript>
 </amp-img>
 
+[/filter]<!-- formats="ads" -->
```

Comment:
Why is this line moved?

Reply:
When I looked at the doc here, at first I got confused https://amp.dev/documentation/components/amp-lightbox/?format=websites The doc that's in between is talking about ads features. It seemed like it should be in the ad filter, specially that the title of the ad section is in the ad filter. So you have actions, then the big title says transparent background are disallowed and in small it's about the ad. But I can put it back where it was if you think it should stay in the web part.